src/pydantic_zarr/v3.py

```diff
@@ -45,13 +52,18 @@
 FillValue = BoolFillValue | IntFillValue | FloatFillValue | ComplexFillValue | RawFillValue
 
 
+class ArrayConfig(TypedDict):
```

Comment:
Is there not an object in `zarr-python` we can import and use here?

Reply:
(I won't be surprised if the answer is no!)

Reply:
there is! https://github.com/zarr-developers/zarr-python/blob/378d5afa7e1cb2ffbed39610a8ec1420d98343d9/src/zarr/core/array_spec.py#L23. I totally forgot writing this....

Reply:
yep, i think it's good to import this instead of redefining it

Reply:
done in https://github.com/zarr-developers/pydantic-zarr/pull/64/commits/605b790246c23cc841be4fcbc05c295957530a90